ross/fluid_flow/fluid_flow.py

```diff
@@ -0,0 +1,400 @@
+        self.length = length
+        self.ltheta = 2.*np.pi
+        self.dz = length / self.n_interv_z
+        self.dtheta = self.ltheta / self.n_interv_theta
+        self.ntotal = self.nz * self.ntheta
+        self.omega = omega
+        self.p_in = p_in
+        self.p_out = p_out
+        self.radius_rotor = radius_rotor
+        self.radius_stator = radius_stator
+        self.xi = xi
+        self.yi = yi
+        self.visc = visc
+        self.rho = rho
+        self.re = np.zeros([self.nz, self.ntheta])
+        self.ri = np.zeros([self.nz, self.ntheta])
+        self.z = np.zeros([1, self.nz])
+        self.xre = np.zeros([self.nz, self.ntheta])
+        self.xri = np.zeros([self.nz, self.ntheta])
+        self.yre = np.zeros([self.nz, self.ntheta])
+        self.yri = np.zeros([self.nz, self.ntheta])
+        self.p_mat = np.zeros([self.nz, self.ntheta])
+        self.bearing_seal_type = -1
+        self.plot_counter = 0
+        self.calculate_coefficients(plot_eccentricity)
+        self.pressure_matrix_available = False
+        self.distance_between_centers = np.sqrt((xi**2 + yi**2))
+        self.difference_between_radius = radius_stator - radius_rotor
+        self.eccentricity_ratio = self.distance_between_centers/self.difference_between_radius
+
+    def calculate_pressure_matrix(self):
+        """This function calculates the pressure matrix
+        """
+        if self.bearing_seal_type == 0:
+            for i in range(self.nz):
+                for j in range(self.ntheta):
+                    self.p_mat[i][j] = ((-3*self.visc*self.omega)/self.difference_between_radius**2)*\
+                                       ((i*self.dz)**2 - (self.length**2)/4)*\
+                                       (self.eccentricity_ratio*np.sin(j*self.dtheta))/\
+                                       (1 + self.eccentricity_ratio*np.sin(j*self.dtheta))**3
+            self.pressure_matrix_available = True
+        return self.p_mat
+
+    def calculate_coefficients(self, plot_cut=False):
+        """This function calculates the constants that form the Poisson equation
+        of the discrete pressure (central differences in the second
+        derivatives). It is executed when the class is instantiated.
+        Parameters
+        ----------
+        plot_cut: bool
+            If True, plots a cut at z=0 and show, so the user can check the
+            internal and external radius as well as eccentricity.
+        """
+        if self.length/self.radius_stator <= 1/8:
+            self.bearing_seal_type = 0
+        elif self.length/self.radius_stator > 4:
+            self.bearing_seal_type = 1
+        else:
+            self.bearing_seal_type = 2
+        if plot_cut:
+            plt.figure(self.plot_counter)
+            self.plot_counter += 1
+        for i in range(self.nz):
+            zno = i * self.dz
+            self.z[0][i] = zno
+            for j in range(self.ntheta):
+                gama = j * self.dtheta
```

Comment:
Use `gamma` for consistency.

Reply:
Did you mean checking if gama is between 0 and 2*pi? If so, ntheta is provided by the user when instantiating, dtheta is calculated automatically based on ntheta, therefore gama can't be wrong.